db/migrate/20210210104407_add_root_repository_id_to_scc_repository.rb

```diff
@@ -0,0 +1,6 @@
+class AddRootRepositoryIdToSccRepository < ActiveRecord::Migration[5.2]
+  def change
+    add_column :scc_repositories, :katello_root_repository_id, :integer, index: true, null: true
```

Comment:
if you install this on a system already using scc manager, the katell_root_repositoryies are not set. Is this an issue? What needs to be done to create the assoication?

Reply:
Good point, I have to think about that.

Reply:
I added a new migration `ConnectKatelloRootRepositoryToSccRepository` to fix this issue.